src/test/e2e/api/admin/tags.e2e.test.ts

```diff
@@ -128,7 +131,7 @@ test('Can tag features', async () => {
     });
 
     await db.stores.tagStore.createTag(removedTag);
-    await db.stores.featureTagStore.tagFeature(featureName, removedTag);
+    await db.stores.featureTagStore.tagFeature(featureName, removedTag, -1337);
```

Comment:
Shouldn't this default to -1337 if we don't send it? Just wondering whether adding this is needed or just makes it more explicit

Reply:
Yes, I was unsure whether this was required by the linter or compiler or just QoL improvement.